pages/projects.js

```diff
@@ -3,18 +3,36 @@ import {
   SectionTitle,
   Container
 } from '../components/shared-styled'
+import {
+  Project,
+  ProjectsContainer
+} from '../components/projects'
 
 export default function Projects() {
     return (
-      <Container>
+      <>
         <Head>t
           <title>Illini Blockchain</title>
-          <meta name="description" content="Generated by create next app" />
-          <link rel="icon" href="/favicon.ico" />
+          <meta name='description' content='Generated by create next app' />
+          <link rel='icon' href='/favicon.ico' />
         </Head>
 
-        <SectionTitle>Projects</SectionTitle>       
+        <SectionTitle>Projects</SectionTitle>
+
+        <ProjectsContainer>
+
+          <Project
+            title={'NFT Mint'}
+            imageSrc={'/illiniBlockchainNFT.svg'}
```

Comment:
we should probably do a screenshot of the cleaner looking one bc the actual render looks wonky based on the device